src/FormTrait.php

```diff
@@ -39,15 +40,16 @@ protected function handleForm(Request $request, $domain, $formType)
 	 * @param Request $request Request with JSON posted data
 	 * @param object $domain
 	 * @param string $formType
+     * @param array $options
 	 * @param boolean $clearMissingFields set to TRUE when you want to validate the whole form (i.e. in POST or PUT) or to FALSE when form contains partial data (i.e. in PATCH)
 	 * @return Form
 	 * @throws BadJSONRequestException
 	 */
-	protected function handleJSONForm(Request $request, $domain, $formType, $clearMissingFields = true)
+	protected function handleJSONForm(Request $request, $domain, $formType, $options = [], $clearMissingFields = true)
```

Comment:
I suggest `$options` be added as the last parameter, so there's no BC break and we can release this as a minor ASAP ;)
BTW, great idea, thanks for the PR!

Reply:
Or.. to maintain BC and still be consistent with `handleForm` order of parameters, add code that would handle `$options` as 3rd (FC) or `$options` as 4th parameter (BC)

Otherwise, I'll accept the PR, but it will be released as major :/

Reply:
Tbh it's up to you if you'd like to push it as a major update, to avoid BC on users who currently uses it, or you want to make it as minor, and move options to be last parameter.
Options were added in mind to be consistent with `handleForm`